components/image/lazyLoad/src/index.scss

```diff
@@ -64,6 +64,12 @@ $aspect-ratios: (
     width: 100%;
   }
 
+  .is-contained {
```

Comment:
I see it more like a modifier, it doesn't seem like a state that can change because of the user behaviour.

Reply:
what do you think @nucliweb ? now that @kikoruiz mentions that, I agree with that...

Reply:
You know, you are walking over the slim line 😅

Feel free